synthetic_data/distinct_generators/datetime_generator.py

```diff
@@ -9,8 +9,8 @@
 def generate_datetime(
     rng: Generator,
     date_format: str,
-    start_date: pd.Timestamp = pd.Timestamp(1920, 1, 1),
```

Comment:
was this intentional?

Reply:
yes, because random datetime will always pass in start and end dates (even if they are none), so we were never setting the start and end date if None was passed in

Reply:
good call

Reply:
do we need to update tests for datetime to account for this change

Reply:
the tests didn't hit on start and end date, so if you want me to add more tests then I can

Reply:
probably good to ensure the inputs are viable to test different conditions